docs/runbooks/deployment/prod-rollback.md

```
@@ -0,0 +1,11 @@
+# Production Rollback
+
+If you need to roll back production to a previous version, **do not just re-run the `post-merge-deploy.yml` workflow against an old tag**.
+
+Re-running the workflow unconditionally runs the `migrate-prod` step, which will attempt to run database migrations. If the bad release included database schema changes, simply deploying the old application code against the new schema will likely cause errors.
+
+### To Rollback Safely:
```

markdownlint-cli2 (0.23.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)